### ISSUE TITLE
Discuss simplyFit architecture and coding standards

### ISSUE THREAD

**Issue by houngj** [houngj/simply_fit#7]
Discussion on what to be used to develop simplyFit and how to go about developing

**Comment by parulkhullar** [comment]
Hello

**Comment by houngj** [comment]
So lets start off what with tools/frameworks. I would like to develop this in Python with the Django web framework. Python because it seems to be becoming a more and more popular language and Django because its super easy to setup and comes with everything we need. Thoughts?

**Comment by parulkhullar** [comment]
Sounds good. I will find some tutorials to refresh my memory.

**Comment by parulkhullar** [comment]
We know the language and the framework we are going to use. Are we using MVC and is that supported by Django?

**Comment by houngj** [comment]
Yeah, we can write MVC in python. But what do you think of trying out the Flux development pattern? I don't see too much complex view controls and need for scalability but it would be cool to try something new!

**Comment by parulkhullar** [comment]
Would flux work with python. It's seems as though facebook used it because it complements ReactJS

**Comment by houngj** [comment]
I'm not sure to be honest... we should do a bit more investigation into Django before we talk about development patterns i guess. Django is definitely atleast a MVC framework though i'm not sure if we can repurpose it for Flux. It may be too early for us to discuss a user friendly front end. Do you think it would be ok to first design the backend API first? After that the API can be easily be used as part of the MVC "Model" or the Flux "Store".

**Comment by houngj** [comment]
Lets put the client side discussion on hold for now and focus on the backend API. We will revisit client side architecture once we have a robust API.
